about.html

```diff
@@ -0,0 +1,117 @@
+            </ul>
+        </div>
+         <!-- menu start-->
+        <nav class="main-menu">
+           <a href="index.html"> <img src="./images/logo.png" alt="logo"> <p> Global <span> Streaming</span> </p></a>
+           <span class="icon">
+            <span class="fa fa-bars"   aria-hidden="true"></span>
+            <span class="fa fa-times"   aria-hidden="true"></span>
+           </span>
+            <ul class="main-menu-ul">
+                
+                <li> <a href="about.html" class="active"> About </a></li>
+                <li><a href="index.html" > Home</a></li>
+                <li> <a href="#"> Live_Streaming </a></li>
+                <li class="disapper"> <a href="index.html#event-categories"> Event </a></li>
+                <li class="disapper"><a  href="#event">See recent</a></li>
+                <li class="disapper"> <a href="#">  Logout </a></li>
+            </ul>
+        </nav>
+    </header>
+<main>
+    <section class="descrption">
+    <h2>GLOBAL . STREAMING</h2>
+    <h2> BEST LIVE STREAMING WEBSITE -2022</h2>
+    <div>
+        <p>Global Streaming is a live-streaming website that allow users to record, 
+            broadcast video and stream it live simultaneously. Also it allows users to broadcast
+             live recordings of videos and audio. Others who have access to the Global Live video streaming,
+              have fueled influencer marketing, entertainment, business marketing, and the way people interact 
+              over the internet.
+             Global Streaming is the best website for sharing and streaming live event in 2022.</p>
+    </div>
+    <p>Please contact us via email for feedback or any other inquiry about Global Streaming</p>
+    <i>globalstreaming2022@gmail.com</i>
+    </section>
+    <section class="recent-events" id="event">
+     <h5>See recent streamed events</h5>
+     <hr>
+     <p> Take a look at three top past events which was streamed live here .....</p>
+     <div class="embeded-video">
+        <div class="video">
+          <iframe width="100%" height="270" src="https://www.youtube.com/embed/c78H-nolqxI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
+        <div class="video">
+          <iframe width="100%" height="270" src="https://www.youtube.com/embed/A_9m5INlV-M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+        </div>
+        <div class="video">
+          <iframe width="100%" height="270" src="https://www.youtube.com/embed/7WU__JQuB8g" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Same for this section, it should have a similar design, the overlay, and the background are wrong and it should have two images with text centered over them instead of 3 videos, this is how it looks on your website:
  ![image](https://user-images.githubusercontent.com/20567503/198016217-b461e9d7-0cd1-45e1-86e3-3a567a3c4399.png)
  This is how it should look: 😁
  ![image](https://user-images.githubusercontent.com/20567503/198016266-872f7790-d4fb-4355-87df-368f39c4e720.png)
  Kindly fix the design. 😄